tss-esapi/tests/integration_tests/structures_tests/lists_tests/tagged_tpm_property_list_tests.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright 2021 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+use std::convert::{TryFrom, TryInto};
+use tss_esapi::{
+    constants::PropertyTag,
+    structures::{TaggedProperty, TaggedTpmPropertyList},
+    tss2_esys::{TPML_TAGGED_TPM_PROPERTY, TPMS_TAGGED_PROPERTY},
+    Error, WrapperErrorKind,
+};
+#[test]
+fn test_valid_conversions() {
+    let expected_tagged_properties: Vec<TaggedProperty> = vec![
+        TaggedProperty::new(PropertyTag::FamilyIndicator, 8u32),
+        TaggedProperty::new(PropertyTag::Level, 12u32),
+        TaggedProperty::new(PropertyTag::HrLoadedMin, 24u32),
+    ];
+
+    let expected_tpml_tagged_tpm_property: TPML_TAGGED_TPM_PROPERTY = expected_tagged_properties
+        .iter()
+        .fold(Default::default(), |mut acc, v| {
+            acc.tpmProperty[acc.count as usize] = TPMS_TAGGED_PROPERTY::from(*v);
+            acc.count += 1;
+            acc
+        });
+
+    let tagged_tpm_property_list_from_vec: TaggedTpmPropertyList = expected_tagged_properties
+        .clone()
+        .try_into()
+        .expect("Failed to convert Vec<TaggedProoperty> into TaggedTpmPropertyList");
+
+    assert_eq!(
+        expected_tagged_properties.len(),
+        tagged_tpm_property_list_from_vec.len(),
+        "Missmatch in 'len()' between the Vec<TaggedProperty> and the TaggedTpmPropertyList(from vec)"
```

Comment:
"mismatch" (also a couple of these below)

Reply:
Fixed